tests/test_qwen_client.py

```
@@ -40,3 +40,30 @@ def fake_urlopen(req: object, timeout: int) -> _FakeResponse:
+            )
+
+        self.assertEqual(captured["timeout"], 60)
+        payload = captured["payload"]
+        self.assertIsInstance(payload, dict)
+        self.assertEqual(payload["model"], "fake-qwen")
+        self.assertEqual(payload["temperature"], 0)
+        self.assertEqual(payload["enable_thinking"], False)
+        self.assertEqual(payload["response_format"], {"type": "json_object"})
+
+    def test_chat_json_object_requires_json_word(self) -> None:
+        with self.assertRaisesRegex(ValueError, "must mention json"):
```

Ruff (0.15.18)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)